AFQ/tasks/segmentation.py

```diff
@@ -260,6 +261,26 @@ def export_bundle_lengths(data_imap,
     return counts_df, dict(sources=bundles_files)
 
 
+@pimms.calc("density_maps")
+@as_file('_desc-density_dwi.npy', include_track=True, include_seg=True)
```

Comment:
Why is this saved into an npy file? Rather than a nifti, I mean.

Also, do we need to save metadata about which volume encodes which bundle?

Reply:
Yes I should save that to the metadata. I guess it should be .nii.gz

Reply:
So right now we are saving a list of the bundles in the metadata. It is in the same order as the volumes. I think this is good enough

Reply:
see: `source=clean_bundles, bundles=list(bundle_dict.keys()))`